i18n/locales/en_us.json

```diff
@@ -50,6 +50,8 @@
   "duplicateThisFrameFramesListTool" :"Duplicate this frame",
   "deleteThisFrameFramesListTool": "Delete this frame",
   "loadingPiskelMessage": "Loading Piskel...",
+  "penSizeChangeTitle": "Pen size TEST 1",
+  "penSizeChangeAt14816": "at 1, 4, 8, or 16 pixels TEST 2",
```

Comment:
The 14816 is kind of odd. Maybe `penSizeChangeAvailableSizes` or something like that?